bin/migration

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+
```

Comment:
Create a folder for migrations. Put this script in that folder. Name this script based on its version/date change. 
For example, migrate-mongodb-20161209.js

Add comments explaining what is included in this migration. Explain the assumptions of the state the database is in. I noticed that you have the language codes you need to update for example, so we should explain that before the database had es and en as codes, now we need to use database objects.